packages/xo-web/src/common/intl/messages.js

```diff
@@ -1520,11 +1520,13 @@ const messages = {
   autoPowerOn: 'Auto power on',
   protectFromDeletion: 'Protect from accidental deletion',
   protectFromShutdown: 'Protect from accidental shutdown',
+  blockMigration: 'Block migration',
+  unblockMigrationTitle: 'Unblock migration',
+  unblockMigrationconfirm: 'This will unblock migration',
```

Comment:
```suggestion
  unblockMigrationConfirm: 'This will unblock migration',
```

Reply:
Done

Reply:
IMHO, this is not a good text for a modal, this would be better:

> Are you sure you want to unblock migration for this VM?

Reply:
And I think we likely want to make it explicit for users that what is listed below is the reasons why it is currently blocked.